### PR TITLE
CT-1065 RxAndroidBle Publish

### PR DESCRIPTION
**Pull request by sonrohan**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -60,3 +60,36 @@ allprojects {
...
+}
+
+def getArtificatVersion = { ->
+    return "1.1"
+}
+
+publishing {
+    publications {
+        app(MavenPublication) {
+            groupId 'com.github.rxandroidble'
```
Comment (ihenchi) [inline build.gradle]: 'com.github.proxyco' instead

File: build.gradle [hunk 2]
```diff
@@ -60,3 +60,36 @@ allprojects {
...
+            artifactId getArtificatId()
+            version getArtificatVersion()
+
+            artifact('./rxandroidble/build/outputs/aar/rxandroidble-debug.aar') {}
```
Comment (ihenchi) [inline build.gradle:81]: why did you choose `debug` here and not `release` ?

File: build.gradle [hunk 3]
```diff
@@ -60,3 +60,36 @@ allprojects {
...
 }
+
+apply plugin: 'maven-publish'
+
+def getArtificatId = { ->
+    return "rxandroidble"
+}
+
+def getArtificatVersion = { ->
+    return "1.1"
```
Comment (ihenchi) [inline build.gradle]: can we sync this with the original library versioning? This way we can keep track

File: README.md [hunk 4]
```diff
@@ -418,3 +418,9 @@ Want to talk about it? Join our discussion on [Gitter](https://gitter.im/RxBLELi
 ## Maintained by
 
 [![Polidea](https://raw.githubusercontent.com/Polidea/RxAndroidBle/master/site/polidea_logo.png "Tailored software services including concept, design, development and testing")](http://www.polidea.com)
+
+## Publish
+- Make sure to set `GPR_USER` and `GPR_API_KEY` environmental variables, where USER is your Github username, and API_KEY is your personal access token for Github.
+- Increment the version of package deploy in build.gradle
```
Comment (ihenchi) [inline README.md]: See comment about getArtificatVersion, I am hoping we don't have to do that

File: build.gradle [hunk 5]
```diff
@@ -60,3 +60,36 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+apply plugin: 'maven-publish'
+
+def getArtificatId = { ->
```
Comment (ihenchi) [inline build.gradle]: typo artifact ?

File: build.gradle [hunk 6]
```diff
@@ -60,3 +60,36 @@ allprojects {
...
+    return "rxandroidble"
+}
+
+def getArtificatVersion = { ->
```
Comment (ihenchi) [inline build.gradle]: artifact

File: build.gradle (hunk 2 above)
Comment (sonrohan) [reply build.gradle:81]: When I built this locally, I only got the debug file. Never even saw a release file.

File: README.md (hunk 4 above)
Comment (sonrohan) [reply README.md]: I think we can get the gradle properties version instead

**Comment by sonrohan** [comment]
Artifact: https://github.com/proxyco/RxAndroidBle/packages/182339

**Review by ihenchi — APPROVED** [review]

**Pull request merged**